app/components/UserList.js

```diff
@@ -63,7 +63,7 @@ const UserList = props => {
               }
               edges {
                 node {
-                  id
+                  rowId
```

Comment:
the rest of this file is still using `node.id`

Reply:
the `id` field is special...see https://facebook.github.io/relay/graphql/objectidentification.htm